test/system/helpers.bash

```diff
@@ -255,6 +255,7 @@ function defer-assertion-failures() {
 # Basic teardown: remove all pods and containers
 function basic_teardown() {
     echo "# [teardown]" >&2
+    echo "$(pidof pasta)"
```

Comment:
This might not help you much. It basically goes nowhere unless the test actually fails. If you want actual output going to the test log, where humans can see it:
```bash
    echo "# $(pidof pasta)" >&3
```
Or if you want more info
```
    ps auxww --forest | grep -v grep | grep -3w pasta | sed -e 's/^/# /' >&3
```